scripts/test.sh

```diff
@@ -9,6 +9,18 @@ set -x
 # Set node options
 export NODE_OPTIONS='--max-old-space-size=5120'
 
+# Clear old code coverage data
+rm -rf coverage
+mkdir -p coverage/packages
+mkdir -p coverage/combined
+
+# Seed the database
+# This is a special "test" which runs all of the seed logic, such as setting up structure definitions
+# On a normal developer machine, this is run only rarely when setting up a new database
+# This test must be run first, and cannot be run concurrently with other tests
```

Comment:
> This test must be run first, and cannot be run concurrently with other tests

*yet*

Got a PR to follow this up that will allow them to be run in parallel thanks to compressed schema 😉